frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.default

```diff
@@ -1,22 +1,13 @@
 # defaults for cuttlefish-host_orchestrator
 
-# The network address to listen on for both HTTP and HTTPS.
+# The network address to listen on.
 # Defaults to localhost only.
 orchestrator_listen_address=0.0.0.0
 #
 # The port on which the orchestrator server should listen on for plain HTTP.
-# Defaults to 2080.
+# Defaults to 2081.
```

Comment:
Do we need this configuration? IIUC, `nginx` would wrap Host Orchestrator, and the actual ports should be accessed by users would set in `/etc/nginx/conf.d/cuttlefish-orchestration.conf`.

Reply:
yes, we should be able to continue accessing the HO service directly.

Reply:
Could you elaborate on here? What cases do we need to access HO directly via `2081`?

Reply:
While chatting with him, heard answer that `2081` would be continuously used for debugging purposes.